src/bluesearch/database/topic.py

```diff
@@ -79,29 +82,21 @@ def request_mesh_from_nlm_ta(nlm_ta: str) -> list[dict] | None:
     )
     footer = "</pre>"
     if not text.startswith(header) or not text.endswith(footer):
-        raise RuntimeError(f"Unexpected response for query\n{url}")
-    text = html.unescape(text[len(header) : -len(footer)]).strip()
+        logger.error(f"Unexpected response for query\n{url}")
+        return None
+    text = html.unescape(text[len(header) - 5 :]).strip()
 
     # Empty text means topic abbreviation was not found. See comment about the
     # parameter "format=text" above.
-    if text == "":
+    if text == "<pre></pre>":
+        logger.error(f"Empty body for query\n{url}")
         return None
 
-    try:
-        content = ElementTree.fromstring(text)
-    except ElementTree.ParseError:
-        # Occurs when the number of results of the research is bigger than one.
-        # It is the case for less than 1 % of the journal from PMC
```

Comment:
A lot of them were giving several results, mainly because we were requesting for title and the request was doing string matching (for example: https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Neurology%22[Title]&report=xml&format=text). In the current situation, we switched to a request with the title abbreviation (NLM TA) - which should be unique (see [here](https://www.nlm.nih.gov/bsd/mms/medlineelements.html#ta)).